transmogrifier.js

```diff
@@ -388,18 +389,49 @@ function getSinkFunction(name) {
     return sinks[name];
 }
 
-function getSchema(name) {
-    // TODO if "name" is a URL, read from the URL (look at main loadManifest())
-    // if "name" is a filepath, read from the filepath
-    // this probably mimics loadManifest but with a schema instead
+async function getSchema(path) {
+    let schema;
+    const http = require('http');
```

Comment:
should the const http and https be outside of this function so that others can use it?

Reply:
It could, but I chose to follow the style of readLocalFileNode() and writeLocalFileNode() where they require it inside the function. Ill do a google search to see if this impacts performance

Reply:
seems like it does (minorly) , ill pull those out

Reply:
Nvm I'm actually going to leave those in. I think the reason readLocalFileNode() requires fs inside the function is because if we were running on a web based platform that module wouldn't be available (because we're not running with node) and we wouldn't want to require it. I know for now getSchema is called like that no matter what platform we're on but keeping it in there makes it easier to separate in the future.